xml/src/main/java/org/ehcache/xml/model/CacheSpec.java

```diff
@@ -46,15 +45,15 @@
 import static java.util.Optional.ofNullable;
 import static java.util.function.Function.identity;
 
-public class BaseCacheSpec implements CacheTemplate {
+public class CacheSpec implements CacheTemplate {
```

Comment:
Should it be abstract? Will it be instantiated directly?

Reply:
It is used directly now after replacing all those interface usages with concrete classes following ur previous comment.

Reply:
Good. Perfect